frontend/package.json

```diff
@@ -6,13 +6,16 @@
     "lint": "eslint . --ext .js",
     "lint:fix": "npm run lint -- --fix",
     "start:dev": "nodemon lib/server.js",
-    "start:prod": "node lib/server.js"
+    "start:prod": "node lib/server.js",
+    "test": "mocha --exit"
   },
   "dependencies": {
     "@aliceo2/web-ui": "1.10.4"
   },
   "devDependencies": {
     "eslint": "6.8.0",
-    "nodemon": "2.0.2"
+    "mocha": "7.1.1",
+    "nodemon": "2.0.2",
+    "puppeteer": "2.1.1"
   }
-}
+}
```

Comment:
Empty line

Reply:
Do you mean an empty line after the file?

Reply:
@john-ghatas yes.

Reply:
@mvegter It is there, but GitHub automatically trims this. You can check locally.